tests/integration/data/test_solver.conf

```diff
@@ -2,7 +2,8 @@
 *
 
 * No iterations
-NITER = 10
+# NITER = 10
+NITER = 1
```

Comment:
What's the purpose of this change? The rest of the commit appears to be to do with getting the sctfcoil and superconductors tests to pass.

Reply:
Reverted